docs/common-coreboot-docs/checkbox.md

```diff
@@ -0,0 +1,222 @@
+    │    ( ) After suspend reference tests (automated)                             │
+    │    ( ) After suspend reference tests (certification blockers only)           │
+    │    ( ) After suspend tests (discrete GPU automated)                          │
+    │    ( ) After suspend tests (discrete GPU manual)                             │
+    │    ( ) After suspend tests (discrete GPU)                                    │
+    │    ( ) After suspend tests (discrete GPU, certification blockers only)       |
+            .
+            .
+            . etc.
+    └──────────────────────────────────────────────────────────────────────────────┘
+    ```
+
+1. Select test suites (`SPACE` - select, `ARROWS` - navigation, `Enter` -
+go to the next test suite), which you want to run. After it all modules should
+be displayed and you can manually uncheck some modules, which you don't want to
+test it.
+
+    ```bash
+    ┌──────────────────────────────────────────────────────────────────────────────┐
+    │[X] + Audio tests                                                             │
+    │[X] + Benchmarks tests                                                        │
+    │[X] + Bluetooth tests                                                         │
+    │[X] + CPU tests                                                               │
+    │[X] + Camera tests                                                            │
+    │[X] + Disk tests                                                              │
+    │[X] + Ethernet Device tests                                                   │
+    │[X] + Firmware tests                                                          │
+    │[X] + Graphics tests                                                          │
+    │[X] + Informational tests                                                     │
+    │[X] + Input Devices tests                                                     │
+    │[X] + Memory tests                                                            │
+    │[X] + Miscellaneous tests                                                     │
+    │[X] + Mobile broadband tests                                                  │
+    │[X] + Non-device specific networking tests                                    │
+    │[X] + Optical Drive tests                                                     │
+    │[X] + Power Management tests                                                  │
+    │[X] + Suspend tests                                                           │
+    │[X] + TPM 2.0 (Trusted Platform Module)                                       │
+    │[X] + Touchpad tests                                                          │
+            .
+            .
+            . etc.
+    └──────────────────────────────────────────────────────────────────────────────┘
+    ```
+
+1. Press `T` to start the testing procedure. You can be asked about hardware in
```

Comment:
This point should be split into two. The first one is about how to start the procedure and the second one is about the hardware listed below.

Reply:
splited